roles/addons/powerman/templates/powerman.conf.j2

```diff
@@ -14,20 +14,23 @@ include "/etc/powerman/ipmipower.dev"
   {% endfor %}
 {% else %}
   {% set range = groups['all'] %}
-{% endif %}
+{%- endif %}
 
-{% for equipment in j2_equipment_groups_list %}
-  {% set hosts = [] %}
-  {% set bmcs = [] %}
-  {% for host in groups[equipment] %}
-    {% if host in range and hostvars[host]['bmc']['name'] is defined %}
-      {{ hosts.append(host) }}
-      {{ bmcs.append(hostvars[host]['bmc']['name']) }}
-    {% endif %}
-  {% endfor %}
-  {% if hosts | length > 0 %}
+{%- for equipment in j2_equipment_groups_list %}
+  {%- set hosts = [] -%}
+  {%- set bmcs = [] -%}
+  {%- for host in groups[equipment] %}
+    {%- if host in range and hostvars[host]['bmc']['name'] is defined %}
+      {{- hosts.append(host) }}
+      {{- bmcs.append(hostvars[host]['bmc']['name']) }}
+    {%- endif %}
+  {%- endfor %}
+  {%- if hosts | length > 0 %}
+
```

Comment:
This adds too much blank lines, can you please remove this one?

Reply:
This is needed to separate different types of equipments, else it is really difficult to read the file.

Reply:
With a blank line at l33, I expect each equipment to be separated from the other with one blank line. Am I wrong?